src/main/java/net/dv8tion/jda/core/entities/Invite.java

```diff
@@ -0,0 +1,275 @@
+        * Returns creation date of this invite.
+        * <br>
+        * <br>This works only for expanded invites and will throw a {@link IllegalStateException} otherwise!
+        *
+        * @throws IllegalStateException if this invite is not expanded
+        *
+        * @return The creation date of this invite
+        *
+        * @see Invite#expand()
+        * @see Invite#isExpanded()
+        */
+    OffsetDateTime getCreationTime();
+
+    /**
+     * Returns an {@link Invite.Guild} object containing info about the invite's guild.
+     *
+     * @return The info about the invite's guild
+     */
+    Guild getGuild();
+
+    /**
+     * Returns the user who created this invite. This may be a fake user.
+     * <br>
+     * <br>This works only for expanded invites and will throw a {@link IllegalStateException} otherwise!
+     *
+     * @throws IllegalStateException if this invite is not expanded
+     *
+     * @return The user who created this invite
+     *
+     * @see Invite#expand()
+     * @see Invite#isExpanded()
+     */
+    User getInviter();
+
+    /**
+     * Returns the {@link net.dv8tion.jda.core.JDA JDA} instance used to create this
+     * @return
+     *      the corresponding JDA instance
+     */
+    JDA getJDA();
+
+    /**
+     * Returns the max age of this invite in seconds.
+     * <br>
+     * <br>This works only for expanded invites and will throw a {@link IllegalStateException} otherwise!
+     *
+     * @throws IllegalStateException if this invite is not expanded
+     *
+     * @return The max age of this invite in seconds
+     *
+     * @see Invite#expand()
+     * @see Invite#isExpanded()
+     */
+    int getMaxAge();
+
+    /**
+    * Returns the max uses of this invite. If there is no limit thius will return {@code 0}.
+    * <br>
+    * <br>This works only for expanded invites and will throw a {@link IllegalStateException} otherwise!
+    *
+    * @throws IllegalStateException if this invite is not expanded
+    *
+    * @return The max uses of this invite or {@code 0} if there is no limit
+    *
+    * @see Invite#expand()
+    * @see Invite#isExpanded()
+    */
+    int getMaxUses();
+
+    /**
+     * Returns how often this invite has been used.
+     * <br>
+     * <br>This works only for expanded invites and will throw a {@link IllegalStateException} otherwise!
+     *
+     * @throws IllegalStateException if this invite is not expanded
+     *
+     * @return The uses of this invite
+     *
+     * @see Invite#expand()
+     * @see Invite#isExpanded()
+     */
+    int getUses();
+
+    /**
+     * Returns weather is invite expanded or not. Expanded invites contain more infos, but they can only be
+     * obtained be {@link net.dv8tion.jda.core.entities.Guild#getInvites() Guild#getInvites()} (requires
+     * {@link net.dv8tion.jda.core.Permission#MANAGE_CHANNEL Permission.MANAGE_CHANNEL}) or
+     * {@link net.dv8tion.jda.core.entities.Channel#getInvites() Channel#getInvites()} (requires
```

Comment:
`{@link net.dv8tion.jda.core.entities.Channel#getInvites() Channel#getInvites()}`
should be
`{@link net.dv8tion.jda.core.entities.Channel#getInvites() Channel.getInvites()}`
Note the `.`